gm4_soul_fragments/data/gm4_soul_fragments/functions/player/spawn_fragment.mcfunction

```diff
@@ -0,0 +1,11 @@
+#@s = player who's died (@a[scores={gm4_sf_died=1..},gamemode=!creative,gamemode=!spectator] at @s)
+#from player/died
+
+scoreboard players add @s gm4_sf_fragments 1
+
+particle minecraft:soul ~ ~1.2 ~ .17 0 .17 .03 3 force
+playsound minecraft:block.chorus_flower.death master @a[distance=..5] ~ ~ ~ .9 2
```

Comment:
Again, don't play on `master`, use `player`. Decrease the volume a bit more might be something you want to do as well, but I didn't test it in game so that's up to you.